src/CLI/clitree/cli-xml/mac.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0"?>
+<!--copyright 2019 Dell, Inc.  
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--> 
+
+<!-- TODO: Update with network instance l2 yang  -->
+
+
+<!DOCTYPE CLISH_MODULE [
+]>
+
+<CLISH_MODULE xmlns="http://www.dellemc.com/sonic/XMLSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
+                        http://www.dellemc.com/sonic/XMLSchema/clish.xsd">
+
+<VIEW name="enable-view">
+    <COMMAND
+        name="show mac"
+        help="show mac"
+        >
+        <ACTION> 
+            builtin="clish_nop"
+        </ACTION>
+    </COMMAND>
+
+<!--====================show mac address-table command======================-->
+    <COMMAND
+        help="show mac address table"
+        name="show mac address-table"
+        >
+        <ACTION> 
+	    python $SONIC_CLI_ROOT/sonic-cli-mac.py get_openconfig_interfaces_interfaces show_mac.j2 show
+        </ACTION>
+    </COMMAND>
+
+<!--====================show mac address-table count command ======================-->
+    <COMMAND
```

Comment:
Better to have the command as #show MAC address-table and "address", "count" and "vlan" as PARAMs. So, this can be extended in future, so that you don't need to write separate command for all the sub-commands of MAC.